package.json

```diff
@@ -3,6 +3,12 @@
   "version": "1.4.2",
   "description": "Open color scheme for web UI",
   "main": "open-color.scss",
+  "files": [
```

Comment:
probably not, they are the same of root open-color.scss right? if yes, dont be needed